modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -103,6 +103,7 @@ const setupiOSOverrides = (editor: Editor) => {
 
 const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, useNodeAnchor: boolean, highlightImmediately: boolean) => {
   const anchorSpec = useNodeAnchor ? getNodeAnchor(editor) : getPointAnchorSpec(editor, e);
+  console.log(anchorSpec.anchor);
```

Comment:
Make sure this gets removed before we merge

Reply:
😅  Removed